packages/sdk-cli/src/models/debuggerHost.ts

```diff
@@ -94,23 +56,19 @@ export async function createDebuggerHost(
     ),
     {
       maxMessageSize: initMessage.maxMessageSize,
-      ...makeHostInfo(hostType),
-    },
-  );
-  host.setInstallHandler(
-    async (bundleData) => {
-      const bundleInfo = await getBundleInfo(bundleData);
-      handleLog(
-        `Sideload received with appID:${bundleInfo.uuid} buildID:${bundleInfo.buildID}`,
-      );
-      return {
-        app: bundleInfo,
-        components: [hostType],
-      };
+      device: hostDescriptor.displayName,
+      // I don't see anywhere that we actually care what this is defined as
+      hostKind: 'device',
     },
-    makeHostCapabilities(hostType, hostProperties),
   );
 
-  await eventPromise(socket, 'close');
-  handleLog('Debugger or relay closed connection');
+  const closePromise = eventPromise(socket, 'close').then(() => {
+    handleLog('Debugger or relay closed connection');
+  });
+
+  return {
+    closePromise,
+    host,
+    handleClose: () => socket.close(),
```

Comment:
The `handle` prefix on the returned function name is confusing. A `handle` function is supposed to be a private function which is triggered by an event from elsewhere, but this one is being returned to outside code. And this isn't even an event handler; it's an action.
```suggestion
    close: () => socket.close(),
```